proxy/src/telemetry/metrics/record.rs

```diff
@@ -84,8 +85,14 @@ impl Record {
                 })
             },
 
-            Event::TlsConfigReloaded =>
+            Event::TlsConfigReloaded(ref when) =>
                 self.update(|metrics| {
+                    let timestamp_secs = when
+                        .duration_since(UNIX_EPOCH)
```

Comment:
Shouldn't this be `duration_since(<proxy start time?>)`?

Reply:
I thought we decided to use absolute timestamps so the control plane can compare them with the proxy start time to determine if the configs had been reloaded since the proxy started?

Reply:
(this is really just to convert the `SystemTime` to a `Duration`, since `Duration`s have an `as_secs` function while `SystemTime`s are completely opaque)

Reply:
> I thought we decided to use absolute timestamps so the control plane can compare them with the proxy start time to determine if the configs had been reloaded since the proxy started?

I agree that's the right thing to do in the long term and if it was already decided to do it in the short term then that's fine with me.

Reply:
the time we want to compare against is also ins seconds since the epoch, so i think it's fine/good to do it this way (we can compare values for a given pod)